frontend/src/e2e-playwright/specs/4_finance/fee-decisions.spec.ts

```diff
@@ -0,0 +1,82 @@
+// SPDX-FileCopyrightText: 2017-2021 City of Espoo
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+import {
+  careArea2Fixture,
+  daycare2Fixture,
+  feeDecisionsFixture,
+  Fixture
+} from 'e2e-test-common/dev-api/fixtures'
+import {
+  insertEmployeeFixture,
+  insertFeeDecisionFixtures,
+  resetDatabase
+} from 'e2e-test-common/dev-api'
+import { newBrowserContext } from '../../browser'
+import config from 'e2e-test-common/config'
+import { Page } from 'playwright'
+import { initializeAreaAndPersonData } from 'e2e-test-common/dev-api/data-init'
+import EmployeeNav from 'e2e-playwright/pages/employee/employee-nav'
+import {
+  FinancePage,
+  FeeDecisionsPage
+} from 'e2e-playwright/pages/employee/finance/finance-page'
+import { waitUntilEqual } from 'e2e-playwright/utils'
+import { employeeLogin } from 'e2e-playwright/utils/user'
+
+let page: Page
+let feeDecisionsPage: FeeDecisionsPage
+
+beforeEach(async () => {
+  await resetDatabase()
+  const fixtures = await initializeAreaAndPersonData()
+  const careArea = await Fixture.careArea().with(careArea2Fixture).save()
+  await Fixture.daycare().with(daycare2Fixture).careArea(careArea).save()
+
+  await insertFeeDecisionFixtures([
+    feeDecisionsFixture(
+      'DRAFT',
+      fixtures.enduserGuardianFixture,
+      fixtures.enduserChildFixtureKaarina,
+      fixtures.daycareFixture.id
+    )
+  ])
+
+  page = await (await newBrowserContext({ acceptDownloads: true })).newPage()
+
+  await insertEmployeeFixture({
+    id: config.financeAdminAad,
+    externalId: `espoo-ad:${config.financeAdminAad}`,
+    email: 'lasse.laskuttaja@test.com',
```

Comment:
Actually, it's `.test` as in `.com` (the spec can be a bit confusing as it's written as `.test.` but that's just the canonical way of writing domains; the last dot is just generally hidden/considered OK to leave out -- `example.com` and `example.com.` are equivalent):
```suggestion
    email: 'lasse.laskuttaja@whatever-can-go-here-and-should-never-resolve.test',
```

Reply:
Alright, should read more carefully 😄

Reply:
Hmmh, looks like these addresses are littered in quite a few places (e.g. dev-data SQL) -- I'm OK with leaving this as tech debt if it looks to be too much effort to update everything.

Reply:
Yeah I won't do completely now, doesn't seem like a top priority right now.